circle.yml

```diff
@@ -0,0 +1,8 @@
+machine:
+  ruby:
+    version: 2.2.1
```

Comment:
Our apps are running on 2.2.1 at the moment. We should upgrade all at the same time.